doc/examples/plot_active_contours.py

```diff
@@ -0,0 +1,84 @@
+"""
+====================================================
+Active Contour Model
+====================================================
+The active contour model is a method to fit open or closed splines to lines or
+edges in an image. It works by minimising an energy that is in part defined by
+the image and part by the spline's shape: length and smoothness. The
+minimization is done implicitly in the shape energy and explicitly in the
```

Comment:
Do you have any good ideas how to explain in a brief statement? If people have heard of implicit/explicit time stepping then they will know what is meant, I think. If they have not, then I'm not sure that they need to know anyway. What do you think?